http.go

```diff
@@ -52,6 +52,9 @@ func (cfg moduleConfig) getReverseProxyDirectorFunc() func(*http.Request) {
 		vs["module"] = vs["module"][1:]
 		r.URL.RawQuery = vs.Encode()
 
+		for k, v := range cfg.HTTP.Headers {
+			r.Header.Add(k, v)
```

Comment:
I only wonder if it maybe might be better to use `Set()` here and accept an array in the module's configuration so that the users could pass more than one value.

Reply:
People could manually construct a multi-value header though, (they are just one header, comma separated right?), so it doesn't add much, and I think it would be confusing for most users.

Reply:
Agreed, there shouldn't be many cases in expexp where one would need to construct multi value headers and for such, it's still doable.